tests/conftest.py

```diff
@@ -213,6 +213,14 @@ def multiblock_all(datasets):
     return pyvista.MultiBlock(datasets)
 
 
+@pytest.fixture()
+def multiblock_all_with_nested_and_none(datasets, multiblock_all):
+    """Return datasets fixture combined in a pyvista multiblock."""
+    # nested = multiblock_all
+    # nested.append(None)
+    return pyvista.MultiBlock([*datasets, None, multiblock_all])
```

Comment:
```suggestion
    return pyvista.MultiBlock([*datasets, None, multiblock_all])
```

Reply:
This has been resolved with [b4b71c0](https://github.com/pyvista/pyvista/pull/6590/commits/b4b71c0718e8dff75fc007f840d756d0e41191e3)

Reply:
Thank you. I should have checked with you before making the suggestion.

Reply:
I probably should have marked it as draft initially. I appreciate the quick turnaround though, so thanks for the initial comments